topics/admin/tutorials/job-destinations/tutorial.md

```diff
@@ -21,17 +21,17 @@ contributors:
   - natefoo
   - bgruening
   - hexylena
+  - nuwang
+  - cat-bro
+  - afgane
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing. (I think `natefoo` and `bgruening` can claim `authorship`, I'm probably an `editing` since I mostly did the GTN conversion of this tutorial)

```
contributions:
  authorship:
    - natefoo
    - bgruening
  editing:
    - hexylena
    - hexylena
  funding: []
  testing: []
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.